iOSVideoApp/iOSVideoApp/views/VideoPlayerView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  VideoPlayerView.swift
+//  iOSVideoApp
+//
+//  Created by Ezra Black on 5/6/20.
+//  Copyright © 2020 Casanova Studios. All rights reserved.
+//
+
+import UIKit
+import Foundation
+import AVFoundation
+
+class VideoPlayerView: UIView {
+    override class var layerClass: AnyClass {
+        return AVPlayerLayer.self
```

Comment:
Remember that now in single-line methods/computed properties, you can remove the `return`. It's fine to leave it, but I kinda like cutting down on the verbosity.

Reply:
I always forget this! I feel if I didn’t put return I would start to lose reasoning of things.. still fresh to the syntax of swift mentally believe it or not.